.gitignore

```diff
@@ -47,6 +47,8 @@ dev/pr-deps/
 dist/
 docs/_site
 docs/api
+sql/docs/docs
```

Comment:
My only main question is, can we do this without introducing a new sql/docs module? can it just live in sql/ and generate output into the top-level docs/ folder?